deformable_ompl/PlanPathQuery[] queries
---
deformable_ompl/PlanPathResult[] results
